bitmap.go

```diff
@@ -1184,3 +1189,106 @@ func FastOr(bitmaps ...*Bitmap) *Bitmap {
 
 	return dst
 }
+
+func (bm *Bitmap) NSplit(externalSize func(start, end uint64) uint64, maxSz uint64) []*Bitmap {
```

Comment:
As there is no `N` involved in the API, I think this could rather be renamed to `Split`?

Reply:
Done